packages/design-system/src/styles/base/_link.scss

```diff
@@ -1,4 +1,5 @@
 /* stylelint-disable selector-class-pattern */
+
```

Comment:
This can be deleted

Reply:
Are you referring to `/* stylelint-disable selector-class-pattern */` here?

Reply:
Oh i'm talking about the new line, just pointing out the last bits of new line clean up